pkg/resourcecreator/leaderelection/leaderelection.go

```diff
@@ -10,27 +10,46 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func Create(source resource.Source, ast *resource.Ast, leaderElection bool) {
+type ElectionMode int
+
+const (
+	ModeEndpoint ElectionMode = iota
```

Comment:
`iota` var ny for meg, så måtte lese meg litt opp på det. Føles litt utypisk Go å ha en slik funksjon, men virket jo morsom og nyttig.